### PR TITLE
feat:add my story line

### PR DESCRIPTION
**Pull request by sagarvasnani**
Please review this, @casey-collab

### CODE REVIEW

File: english/daughter/file.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿
```
Comment (casey-collab) [inline english/daughter/file.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/daughter/file.md (hunk 1 above)
Comment (casey-collab) [inline english/daughter/file.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by sagarvasnani** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by sagarvasnani** [comment]
Please review this, @casey-collab

File: english/dream/swim/swim.md [hunk 2]
```diff
@@ -11,3 +11,5 @@ A pain shoots up your legs. You glance down to where your feet should be only to
 flipper wrapped tightly in the ends of licorice whips.
 
 You are trapped. Your dream is going horribly wrong. You have been ensnared by... 
+
+[Snake](../dream/livethedream/swim/snake.md)
```
Comment (casey-collab) [inline english/dream/swim/swim.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/dream/swim/Snake/snake.md [hunk 3]
```diff
@@ -0,0 +1 @@
+You have been ensnared by a snake. You have been tied up by the snake's body.
```
Comment (casey-collab) [inline english/dream/swim/Snake/snake.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/daughter/file.md (hunk 1 above)
Comment (casey-collab) [inline english/daughter/file.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/dream/swim/swim.md [hunk 4]
```diff
@@ -11,3 +11,5 @@ A pain shoots up your legs. You glance down to where your feet should be only to
 flipper wrapped tightly in the ends of licorice whips.
 
 You are trapped. Your dream is going horribly wrong. You have been ensnared by... 
+
+[Snake](../dream/livethedream/swim/snake/snake.md)
```
Comment (casey-collab) [inline english/dream/swim/swim.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by sagarvasnani** [comment]
Please review this, @casey-collab

File: english/dream/swim/Snake/snake.md (hunk 3 above)
Comment (casey-collab) [inline english/dream/swim/Snake/snake.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**